src/images/mongo.rs

```diff
@@ -0,0 +1,76 @@
+use std::collections::HashMap;
+
+use crate::{Container, Docker, Image, WaitForMessage};
+
+const CONTAINER_IDENTIFIER: &'static str = "mongo";
+const DEFAULT_TAG: &'static str = "latest";
+
+#[derive(Debug, Default, Clone)]
+pub struct MongoArgs;
+
+impl IntoIterator for MongoArgs {
+    type Item = String;
+    type IntoIter = ::std::vec::IntoIter<String>;
+
+    fn into_iter(self) -> <Self as IntoIterator>::IntoIter {
+        vec![].into_iter()
+    }
+}
+
+#[derive(Debug)]
+pub struct Mongo {
+    tag: String,
+    arguments: MongoArgs,
+}
+
+impl Default for Mongo {
+    fn default() -> Self {
+        Mongo {
+            tag: DEFAULT_TAG.to_string(),
```

Comment:
Can you make the `DEFAULT_TAG` something that doesn't change please?

Having these kind of rolling tags is asking for trouble further down the line where things break because the image is changing.